buf.yaml

```diff
@@ -0,0 +1,8 @@
+version: v1
+name: buf.build/prometheus/client_model
```

Comment:
This won't work until this org and repository is in place, do you want to create it and take ownership of it?

Reply:
I've asked @roidelapluie to create directly as he is a prometheus maintainer.

Reply:
FYI, I've reserved this under out bot user for now, please reply here or reach out on our Slack if you want to take over ownership of this organization. Also CC @brancz, @bwplotka, they might be interested in this effort too.

Reply:
We'll want to update this to point to the managed repo for a smooth transition: `buf.build/prometheus/client-model`. The BSR does not allow `_` in repository names.